src/App.jsx

```diff
@@ -1,12 +1,27 @@
-import { useState } from "react";
+import React, { useState } from "react";
+import Statistics from "./Statistics";
+import Button from "./Button";
+import StatisticLine from "./StatisticLine";
 
 const App = () => {
-  // save each button click in its own state
   const [good, setGood] = useState(0);
   const [neutral, setNeutral] = useState(0);
   const [bad, setBad] = useState(0);
 
-  return <div>code here</div>;
+  return (
+    <div>
+      <h1>Give Feedback</h1>
+      <Button onClick={() => setGood(good + 1)} text="Good" />
```

Comment:
Be careful with this setter implementation. In this case it's better to use a callback to retrieve previous value and update it.

FIX: fix setter and extract it to a handler

```js
const handleGoodIncrease = () => {
  setGood((prevGoodValue) => {
    return prevGoodValue + 1
  })
}

//...

<button onClick={handleGoodIncrease}>good</button>
```